src/IRac.cpp

```diff
@@ -54,6 +54,9 @@ bool IRac::isProtocolSupported(const decode_type_t protocol) {
 #if SEND_DAIKIN160
     case decode_type_t::DAIKIN160:
 #endif
+#if SEND_DAIKIN176
```

Comment:
This appears to be a diff/PR that we have already merged into master.
Can you please `rebase` this branch/PR to an up-to-date copy of the `master` branch?
e.g.
```
git checkout master
git pull
git checkout pasna-patch-3  # Or what ever your local branch is called.
git rebase master  # Follow the instructions if there are conflicts etc.
# Once fully merged etc and ready.
git push --force
```

That should get your client up to sync etc.